### PR TITLE
Inventory and Ingredient creation

### PR DESCRIPTION
**Pull request by AliceJen17**
Inventory Update and Delete methods are still under the testing.

### CODE REVIEW

File: src/main/resources/db/migration/V0__base_version.sql [hunk 1]
```diff
@@ -84,7 +84,7 @@ create table inventory (
    inventory_id integer NOT NULL AUTO_INCREMENT,
    ingredient_id integer ,
    purchase_date timestamp,
-   expiration_date timestamp,
+   expiration_date timestamp  NOT NULL,
```
Comment (ShanchaoLi1992) [inline src/main/resources/db/migration/V0__base_version.sql]: Remove this change.

File: src/main/java/com/example/myKitchenManager/security/Util.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    }
+
+//    public static <T> T findOneById(CrudRepository<T, Long> repo, Long id) {
+//        return repo.findOne(id);
+//    }
+//
+//    public static <T> T save(CrudRepository<T, Long> repo, T entity) {
+//        return repo.save(entity);
+//    }
+}
```
Comment (ShanchaoLi1992) [inline src/main/java/com/example/myKitchenManager/security/Util.java]: What's the use of this Util class?

File: src/main/resources/db/migration/V0__base_version.sql (hunk 1 above)
Comment (AliceJen17) [reply src/main/resources/db/migration/V0__base_version.sql]: It doesn't work. I am using cloud DB temporarily.

File: src/main/java/com/example/myKitchenManager/security/Util.java (hunk 2 above)
Comment (AliceJen17) [reply src/main/java/com/example/myKitchenManager/security/Util.java]: The helper class is helping me debugging right now, I might remove it later. let's see.

**Pull request closed without merging**